test/controllers/votes_controller_test.rb

```diff
@@ -0,0 +1,94 @@
+require "test_helper"
+
+describe VotesController do
+  describe 'create' do
+    it 'can add a new vote' do
+      work = works(:test_album)
+      user = User.new(name: 'tester')
+
+      perform_login(user)
+
+      expect {
+        post work_votes_path(work)
+      }.must_differ 'work.votes', 1
+
+      vote = Vote.find_by(user_id: user.id)
```

Comment:
Since you're creating a new user you'll also need to find them again in the DB since now they have an id number

```suggestion
       user = User.find_by(name: user.name)
      vote = Vote.find_by(user_id: user.id)
```